src/Components/Server/src/BlazorPack/SequenceOfT.cs

```diff
@@ -443,7 +443,7 @@ private void ClearReferences(int startIndex, int length)
                 // If we store references, clear them to allow the objects to be GC'd.
                 if (!IsValueTypeElement)
                 {
-                    this.AvailableMemory.Span.Slice(startIndex, length).Fill(default);
+                    this.AvailableMemory.Span.Slice(startIndex, length).Clear();
```

Comment:
The BlazorPack sources are shared-source with `MessagePack-CSharp`, so to update this the process would be:

 * Have the desired update made in the `MessagePack-CSharp` repo
 * Update the corresponding sources in this repo to match

Reply:
Thanks for the info --> https://github.com/neuecc/MessagePack-CSharp/pull/848

> You've submitted a PR that modifies code that is shared with https://github.com/dotnet/runtime .

The bot is somewhat misleading as it speaks about "runtime"-repo. Humans are still smarter :wink: